docker/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN apt-get -y update && \
 RUN groupadd --gid 1000 opensearch-benchmark && \
     useradd -d /opensearch-benchmark -m -k /dev/null -g 1000 -N -u 1000 -l -s /bin/bash benchmark
 
-RUN python3 -m pip install h5py==3.10.0; if [ -z "$VERSION" ] ; then python3 -m pip install opensearch-benchmark ; else python3 -m pip install opensearch-benchmark==$VERSION ; fi
+RUN if [ -z "$VERSION" ] ; then python3 -m pip install opensearch-benchmark ; else python3 -m pip install opensearch-benchmark==$VERSION ; fi
```

Comment:
why are we removing h5py installation inside dockerfile? Is it because it is already a dependency of OSB installation and will be present as part of installation?

Reply:
Yes, h5py is already included in setup.py and is already a dependency during the OSB installation. If we decide to pin to a version of a dependency, we should do it in setup.py and not the Dockerfile in order to prevent future conflicts.